gatsby-node.js

```diff
@@ -37,4 +42,19 @@ exports.createPages = async ({ graphql, actions }) => {
       },
     });
   });
+
+  projectArticleQuery.data.allContentfulProgetti.nodes.forEach((project) => {
+    const courseSlug = project.project_category[0].slug;
+    const slug = project.titolo.split(" ").join("-").toLowerCase();
+
+    createPage({
+      path: `/progetti/${courseSlug}/${slug}`,
+      component: path.resolve(`./src/template/ProjectArticle.tsx`),
+      context: {
+        slug: `/progetti/${courseSlug}/${slug}`,
```

Comment:
A cosa ti serve questo slug dopo?

Reply:
In realtà a nulla, pensavo di doverlo passare a qualche Schema ma me lo sono immaginato

Reply:
No per lo schema conviene che apri una `issue` dove per creare un customHook che ci ritorni questo valore. 
```
 const pathname = React.useMemo(() => {
    if (hasMounted) return window.location.pathname;
  }, [hasMounted]);
```